### DISCUSSION TITLE
Switching camera (LensFacing.Back/LensFacing.Front) after startScan()

### DISCUSSION THREAD

**Discussion by simonetassi** [capawesome-team/capacitor-mlkit#156] [Q&A] [answered]
I was wondering if it is possible to switch to front camera while the scanner is open (opened with lensFacing option equal to LensFacing.Back), for example via button click. I don't see this option documented. Thank to anybody will help!

**Comment by robingenz** [comment, marked as answer]
The easiest way at the moment is to call `stopScan()` and then `startScan({ lensFacing: ... })` again.

**Reply by simonetassi** [reply]
Thanks for your answer! I thought it would have led to a bad ux but it doesn't affect it too much.
